### PR TITLE
(docs)Updated meta description

### PR DESCRIPTION
**Pull request by lampa-da**
This PR introduces a range of updates to Cohere's models, including new model sizes, improved performance, and enhanced capabilities.

- **New Model Sizes and Token Lengths:** Representation models are now available in `medium-20220217` and `large-20220217` sizes, with an updated `small-20220217` version and the previous `small` model accessible as `small-20211115`. The maximum tokens length per text has increased from 512 to 1024, with a recommendation to keep text lengths below 128 tokens for optimal performance.
- **Extremely Large (Beta) Model:** The `xlarge-20220301` model, the biggest and most performant generation model, is introduced. It outperforms the previous `large` model on various downstream tasks, including sentiment analysis, named entity recognition (NER), and common sense reasoning.
- **Classification Endpoint:** Classification is now available via the classification endpoint, powered by the `small` and `medium` generation models, supporting few-shot classification. Support for Choose Best will be deprecated by May 18th.
- **Finetuning Availability and Policy Updates:** Finetuning is now available to all users, along with updated policies and agreements.
- **Updated Generation Models:** The `small`, `medium`, and `large` generation models have been updated, providing better performance with a new, high-quality dataset.
- **New Extremely Large Model:** The `xlarge` model has been improved, offering better generation quality and a 4x faster prediction speed. It now supports a maximum token length of 2048 tokens and frequency and presence penalties.
- **Updated Small, Medium, and Large Generation Models:** These models have been updated for improved stability and effectiveness, with fixes to quantization and penalties.
- **New and Improved Generation and Representation Models:** The `small`, `medium`, and `large` generation and representation models have been retrained, with updated representation models supporting contexts up to 4096 tokens (previously 1024 tokens). Text lengths below 512 tokens are recommended for optimal performance.
- **Optional Model Parameter:** APIs now offer default endpoints for ease of use, and users can specify models for more control by adding a model param in the request.
- **Moderate (Beta):** The Moderate (Beta) model is introduced for classifying harmful text across various categories, with performance improvements expected over time.
- **Pricing Update and New Dashboard UI:** The update includes improved team features, model accuracy metrics, and flat-rate pricing for an enhanced user experience.
- **Co.classify Powered by Representational Model Embeddings:** The Co.classify endpoint now serves few-shot classification tasks using embeddings from the Representational model for the `small`, `medium`, and `large` default models.
- **New Logit Bias Experimental Parameter:** Generative models now have the option to use the `logit_bias` parameter to prevent unwanted token generation or incentivize desired tokens.
- **Improved Developer Docs:** The developer documentation has been updated with interactive tutorials, better information architecture, and a UI refresh for a sleeker experience.
- **New and Improved Medium and Extremely Large Models:** The PR introduces improvements to current models and a new Beta Model (Command) with advanced capabilities.
- **Model Sizing Update and Improvements:** Cohere's generative models are being consolidated, with only the Medium (focused on speed) and X-Large (focused on quality) models being served. Support for the Medium embedding model is being discontinued.
- **Multilingual Text Understanding Model and Language Detection:** Cohere's new multilingual model, `multilingual-22-12`, offers a unified approach to searching across 100 languages with a single index, simplifying multilingual search.
- **Command Model Nightly Available:** Nightly versions of the Common models are now available, offering improved generative capabilities with weekly performance enhancements.
- **Command R+:** Command R+, Cohere's newest and most performant large language model, is optimized for conversational interaction and long-context tasks, recommended for high-performance and accuracy use cases.
- **Multilingual Support for Co.classify:** The co.classify endpoint now supports the `multilingual-22-12` model for multilingual embedding.
- **Model Name Changes:** Cohere is updating model names for consistency and simplicity, allowing users to call models via the API and SDK with the new names.
- **Updated Maximum Document Length for co.rerank:** The maximum number of documents calculation for co.rerank has been updated, and the endpoint will error if `len(documents) * max_chunks_per_doc >10,000`, with `max_chunks_per_doc` set to 10 as default.
- **Command Model Updates:** The Command model has been updated, with improvements in code, conversational capabilities, and reasoning skills on various tasks.
- **Command Model Updates and Finetuning SDK:** The Command model has been updated, with improvements in reasoning and conversational capabilities. The finetuning SDK has also been updated, and Okta OIDC support is added for seamless SSO.
- **co.chat() and Chat + RAG Playground:** Cohere is releasing co.chat() and the Chat + RAG Playground, offering a groundbreaking generative AI experience with updated command models and new rate limits.
- **Connectors for RAG, Expanded Fine-tuning, Embed Jobs API, and Multi-language SDK Support:** Cohere introduces new features, including <co: 2

### CODE REVIEW

File: fern/pages/deployment-options/cohere-on-aws/amazon-bedrock.mdx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+slug: docs/amazon-bedrock
 hidden: false
-description: "This document provides prerequisites for using Cohere's models on Amazon Bedrock, including subscribing to the models, installing the AWS Python SDK, and configuring authentication credentials. It also includes code snippets to invoke Cohere's embed and text generation models using Bedrock."
-image: "../../../assets/images/8dbcb80-cohere_meta_image.jpg"  
-keywords: "Cohere on AWS, language models on AWS, Amazon Bedrock, Amazon SageMaker"
-
-createdAt: "Thu Feb 01 2024 18:08:37 GMT+0000 (Coordinated Universal Time)"
-updatedAt: "Thu May 30 2024 16:00:53 GMT+0000 (Coordinated Universal Time)"
+description: >-
+  Get started with Cohere's models on Amazon Bedrock: prerequisites, AWS Python
```
Comment (mrmer1) [inline fern/pages/deployment-options/cohere-on-aws/amazon-bedrock.mdx:6]:
```suggestion
This document provides a guide for using Cohere's models on Amazon Bedrock.```

File: fern/pages/deployment-options/cohere-on-aws/amazon-bedrock.mdx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  SDK setup, authentication, and code snippets.
```
Comment (mrmer1) [inline fern/pages/deployment-options/cohere-on-aws/amazon-bedrock.mdx:7]:
```suggestion
```

File: fern/pages/fine-tuning/classify-fine-tuning/classify-starting-the-training.mdx [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+slug: docs/classify-starting-the-training
 hidden: false
-description: "This document provides a guide on how to start training a fine-tuning model for Classification using both the Web UI and the Python SDK, including steps for creating a model, uploading data, previewing data, starting training, and calling the fine-tuned model."
-image: "../../../assets/images/3fe7824-cohere_meta_image.jpg"  
-keywords: "classification models, fine-tuning language models, fine-tuning"
-
-createdAt: "Fri Nov 10 2023 18:14:01 GMT+0000 (Coordinated Universal Time)"
-updatedAt: "Thu Jun 13 2024 13:10:55 GMT+0000 (Coordinated Universal Time)"
+description: >-
+  Fine-tune classification models with Cohere's Web UI or Python SDK, using
```
Comment (mrmer1) [inline fern/pages/fine-tuning/classify-fine-tuning/classify-starting-the-training.mdx:6]:
```suggestion
Fine-tune classification models with Cohere's Web UI or Python SDK using custom datasets. ```

File: fern/pages/fine-tuning/classify-fine-tuning/classify-starting-the-training.mdx [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+  custom datasets for tailored predictions.
```
Comment (mrmer1) [inline fern/pages/fine-tuning/classify-fine-tuning/classify-starting-the-training.mdx:7]:
```suggestion
```

File: fern/pages/fine-tuning/fine-tuning-with-the-cohere-dashboard.mdx [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+slug: docs/fine-tuning-with-the-cohere-dashboard
 hidden: false
-description: "Customers can use the Cohere Web UI to prepare and validate data for fine-tuning jobs, making it easier for those who prefer a web interface over programming with the Python SDK. After uploading a dataset, users can start the fine-tuning job and track its progress through various status messages."
-image: "../../assets/images/da7d0fa-cohere_meta_image.jpg"  
-keywords: "fine-tuning, fine-tuning large language models"
-
-createdAt: "Wed Nov 15 2023 21:36:22 GMT+0000 (Coordinated Universal Time)"
-updatedAt: "Thu Jun 13 2024 00:39:00 GMT+0000 (Coordinated Universal Time)"
+description: >-
+  Kick off your fine-tuning journey with Cohere's dashboard—a seamless web
```
Comment (mrmer1) [inline fern/pages/fine-tuning/fine-tuning-with-the-cohere-dashboard.mdx:6]:
```suggestion
 Use the Cohere Web UI to start the fine-tuning jobs and track the progress.```

File: fern/pages/fine-tuning/fine-tuning-with-the-cohere-dashboard.mdx [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
+  interface for customers to initiate and manage their fine-tuning jobs.
```
Comment (mrmer1) [inline fern/pages/fine-tuning/fine-tuning-with-the-cohere-dashboard.mdx:7]:
```suggestion
```

File: fern/pages/fine-tuning/fine-tuning-with-the-python-sdk.mdx [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
+slug: docs/fine-tuning-with-the-python-sdk
 hidden: false
-description: "Customers can fine-tune models using the Cohere Python SDK programmatically, uploading datasets with training and evaluation data before starting a fine-tuning job, and monitoring the results through email notifications and the Dashboard Web UI."
-image: "../../assets/images/782e60c-cohere_meta_image.jpg"  
-keywords: "python, fine-tuning, fine-tuning large language models"
-
-createdAt: "Fri Nov 10 2023 18:29:56 GMT+0000 (Coordinated Universal Time)"
-updatedAt: "Thu May 09 2024 02:54:41 GMT+0000 (Coordinated Universal Time)"
+description: >-
+  Fine-tune language models via the Web UI, API, or Python SDK using custom
```
Comment (mrmer1) [inline fern/pages/fine-tuning/fine-tuning-with-the-python-sdk.mdx:6]:
```suggestion
Fine-tune models using the Cohere Python SDK programmatically and monitor the results through the Dashboard Web UI.```

File: fern/pages/fine-tuning/fine-tuning-with-the-python-sdk.mdx [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
+  datasets to adapt models for specific use cases.
```
Comment (mrmer1) [inline fern/pages/fine-tuning/fine-tuning-with-the-python-sdk.mdx:7]:
```suggestion
```

File: fern/pages/fine-tuning/rerank-fine-tuning/rerank-improving-the-results.mdx [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
+slug: docs/rerank-improving-the-results
 hidden: false
-description: "This document provides tips for achieving the best fine-tuned rerank model, focusing on refining data quality through text cleaning, diversity of queries and documents, hard negatives, text length, and including a high-quality test set. It also offers a troubleshooting guide for fine-tuned models."
-image: "../../../assets/images/55d219e-cohere_meta_image.jpg"  
-keywords: "fine-tuning, fine-tuning language models, rerank"
-
-createdAt: "Thu Nov 16 2023 02:59:09 GMT+0000 (Coordinated Universal Time)"
-updatedAt: "Fri Mar 15 2024 04:42:26 GMT+0000 (Coordinated Universal Time)"
+description: >-
+  Fine-tune your rerank model with diverse, cleaned data, and a robust test set
```
Comment (mrmer1) [inline fern/pages/fine-tuning/rerank-fine-tuning/rerank-improving-the-results.mdx:6]:
```suggestion
Tips for achieving the best fine-tuned rerank model and troubleshooting guide for fine-tuned models.```

File: fern/pages/fine-tuning/rerank-fine-tuning/rerank-improving-the-results.mdx [hunk 10]
```diff
@@ -1,14 +1,14 @@
...
+  for optimal results.
```
Comment (mrmer1) [inline fern/pages/fine-tuning/rerank-fine-tuning/rerank-improving-the-results.mdx:7]:
```suggestion
```

File: fern/pages/changelog/2022-02-22-larger-representation-models.mdx [hunk 11]
```diff
@@ -1,8 +1,11 @@
 ---
-title: "Larger Representation Models"
-slug: "changelog/larger-representation-models"
-type: "added"
-createdAt: "Tue Feb 22 2022 19:05:00 GMT+0000 (Coordinated Universal Time)"
+title: Larger Representation Models
+slug: changelog/larger-representation-models
+type: added
+createdAt: 'Tue Feb 22 2022 19:05:00 GMT+0000 (Coordinated Universal Time)'
```
Comment (lampa-da) [inline fern/pages/changelog/2022-02-22-larger-representation-models.mdx:5]:
```suggestion
title: Larger Representation Models
slug: changelog/larger-representation-models
```

File: fern/pages/text-generation/tokens-and-tokenizers.mdx [hunk 12]
```diff
@@ -1,14 +1,13 @@
...
-image: "../../assets/images/5d536ac-cohere_meta_image.jpg"  
-keywords: "language model tokens, natural language processing"
-
-createdAt: "Thu Feb 29 2024 18:14:01 GMT+0000 (Coordinated Universal Time)"
-updatedAt: "Thu May 23 2024 05:39:13 GMT+0000 (Coordinated Universal Time)"
+title: Tokens and Tokenizers
+slug: docs/tokens-and-tokenizers
+hidden: false
+description: >-
+  This document explains that language models use tokens instead of characters or bytes.
```
Comment (mrmer1) [inline fern/pages/text-generation/tokens-and-tokenizers.mdx:6]:
```suggestion
  This document describes how to use the tokenize and detokenize API endpoints.
```

**Review by mrmer1 — APPROVED** [review]
